### ISSUE TITLE
Non tokenized elements at the beginning of paragraph

### ISSUE THREAD

**Issue by matyaskopp** [hpreki/ParCzech4AT#2]
I have found the situation when the udpipe annotation fails. It happens when a non-tokenized element is at the beginning of a paragraph (page break, note or incident):

page break sample:
```XML
               <seg xml:id="NRSITZ_020_00040_d2e2960">
                  <pb n="109"/>
            Herr Kollege Zweytick! Sie werden es vielleicht noch ein paar Wochen aushalten. Herr Kollege Maitz! Sie werden es vielleicht noch ein paar Monate aushalten. Aber Sie werden sich nicht mehr im Ennstal sehen lassen können, wenn Sie diese Art der Politik des Kopf-in-den-Sand-Steckens weiterhin fortführen, wenn Sie diese Art der Politik nicht ändern.
          </seg>
```

note sample:
```XML
            <u who="#PAD_02840" xml:id="_d7e7027" ana="ucat:regular">
               <seg xml:id="NRSITZ_020_00040_d2e2206">
                  <note type="procedural">(fortsetzend) </note> Meine Damen und Herren! Heute haben Sie Gelegenheit, diese irre Regelung aufzuheben, bis sich wirklich die Expertenkommission einig geworden ist, was nun mit diesen Fällen zu geschehen hat. <kinesic type="applause">
                     <desc>(Beifall bei den Freiheitlichen.) </desc>
                  </kinesic>
               </seg>
               <note type="time">
                  <time when="1996-10-02T15:30:00+02:00">15:30</time>
               </note>
            </u>
```

In the case of `<pb>`: it does not make sense to have a page break at the paragraph edge. So I suggest fixing data and placing such  `<pb>`s outside `<seg>` element.

Placing even edge notes+incidents (and the other) outside will fix this issue, I hope. Keep in mind that empty segments should be removed then.

**Comment by hpreki** [comment]
@matyaskopp  I followed your advice and moved `<seg>` initial non-tokenized elements out of `<seg/>`.
This affects: `<pb> <note> <vocalic> <kinetic> and <incident>`

The updated TEI is now a proper part of this repo - and can be viewed  under TEST_ParlaMint-AT. 

I re-ran udpipe and the results are mixed: the overall number of faulty files dropped from  48 to 20. 
I.e. for 28 files the number auf validation errors now is 0. 
For the rest of the files the number validation errors either dropped significantly but for other it just stayed the same. 

Old and new error-stats can now be compared in the folder `/ERRORS`

```
~/ParCzech4AT/ERRORS$ head -n 3 errors_anaTEST_2022-09-2*
==> errors_anaTEST_2022-09-20_NUMber_errors_per_file.csv <==
 100152	ParlaMint-AT_2007-12-04-023_XXIII_NRSITZ_00040.ana.xml
  42648	ParlaMint-AT_2000-07-05-021_XXI_NRSITZ_00032.ana.xml
  40921	ParlaMint-AT_2002-01-31-021_XXI_NRSITZ_00092.ana.xml
...
==> errors_anaTEST_2022-09-22_NUMber_errors_per_file.csv <==
 100316	ParlaMint-AT_2007-12-04-023_XXIII_NRSITZ_00040.ana.xml
  21336	ParlaMint-AT_2001-07-05-021_XXI_NRSITZ_00075.ana.xml
  17584	ParlaMint-AT_1997-12-12-020_XX_NRSITZ_00105.ana.xml
  ...
```
I.e. the changes _had_ some obvious positive effects, but not on _all_ files (the worst file `ParlaMint-AT_2007-12-04-023_XXIII_NRSITZ_00040.ana.xml` now even has a few errors more) and it did not get rid of all errors. 

(Note: udpipe2 was run without the new `--try2fix-spaces` - my fault - but I do not think that posed the problem, because apart from a bug in 2 files I fixed the `space-around-<pb>`-issue in the input data. But I will re-test with  `--try2fix-spaces` as well)
